pkgs/development/compilers/ocaml/generic.nix

```diff
@@ -10,15 +10,17 @@ let
   safeX11 = stdenv: !(stdenv.isAarch32 || stdenv.isMips);
 in
 
-{ stdenv, fetchurl, ncurses, buildEnv
+{ stdenv, fetchurl, ncurses, buildEnv, libunwind
 , libX11, xorgproto, useX11 ? safeX11 stdenv && !stdenv.lib.versionAtLeast version "4.09"
 , aflSupport ? false
 , flambdaSupport ? false
+, spaceTimeSupport ? false
 }:
 
 assert useX11 -> !stdenv.isAarch32 && !stdenv.isMips;
 assert aflSupport -> stdenv.lib.versionAtLeast version "4.05";
 assert flambdaSupport -> stdenv.lib.versionAtLeast version "4.03";
+assert spaceTimeSupport -> stdenv.lib.versionAtLeast version "4.04";
```

Comment:
Both other “xxSupport” options change the name of the derivation. Should “spaceTime” somehow appear in the compiler name when enabled?

Reply:
fixed

Reply:
Sorry to ask: are you sure it is fixed?

Reply:
oh I had forgotten to commit, sorry